sdk/core/core-http/Changelog.md

```diff
@@ -1,8 +1,11 @@
 # Changelog
 
-## 1.0.0-preview.5 - TBD
+## 1.0.0-preview.6 - Coming Soon
 
-- Removed error type `ResponseBodyNotFoundError`. Use cases for it were removed.
+- Removed error type `ResponseBodyNotFoundError` that was introduced in the previous preview. Use cases for it were removed.
+- Placeholder for New logging support via the new logger package
+- Placeholder for the fix that parses error responses using default mappers
+- More
 
 ## 1.0.0-preview.4 - 2019-10-07
```

Comment:
Is it worth indicating the double release so that it doesn't just jump to preview.6?

Reply:
yup, but in a separate PR. This one took hours to compete all the checks, not pushing another commit to this one :)

Reply:
Fully understand, sounds good!